tests/foreman/ui/test_discoveredhost.py

```diff
@@ -141,6 +135,234 @@ def setUpClass(cls):
         cls.discovery_auto.value = 'True'
         cls.discovery_auto.update({'value'})
 
+        # Create a new Life-Cycle environment
```

Comment:
So same approach can be applied here. Just create organization and location in setup and pass it to that helper